website/docs/reference/api-tokens-and-client-keys.mdx

```diff
@@ -57,13 +56,16 @@ Do **not** use admin tokens for:
 Support for scoped admin tokens with more fine-grained permissions is currently in the planning stage.
 
 
-### Project level access tokens
-
-All project members can see any `CLIENT` (server-side client token) and `FRONTEND` (client-side client token) respectively.
+### API token visibility
+:::info project-level visibility
+Project-level visibility and access to API tokens was introduced in Unleash 4.22.
+:::
+By default, only admin users can create API tokens, and only admins can see their values.
+However, any [client](#client-tokens client tokens) and [front-end tokens](#front-end-tokens) that are applicable to a project, will also be visible to any members of that project that have the `READ_PROJECT_API_TOKEN` permission (all project members by default).
+Similarly, any project members with the `CREATE_PROJECT_API_TOKEN` permission can also create client and front-end tokens for that specific project ([how to create project API tokens](../how-to/how-to-create-project-api-tokens.mdx)).
```

Comment:
This is great, but I think it should be **before** the admin tokens section. Because it applies to tokens in general. The other sections starting at admin tokens are all about specific types of tokens. What do you think?

Also, I think we should add some more spacing in here to both break up the text for end users and to prevent prettier from formatting this weirdly and breaking the admonition later.

```suggestion
### API token visibility

:::info project-level visibility

Project-level visibility and access to API tokens was introduced in Unleash 4.22.

:::

By default, only admin users can create API tokens, and only admins can see their values.

However, any [client](#client-tokens client tokens) and [front-end tokens](#front-end-tokens) that are applicable to a project, will also be visible to any members of that project that have the `READ_PROJECT_API_TOKEN` permission (all project members by default).

Similarly, any project members with the `CREATE_PROJECT_API_TOKEN` permission can also create client and front-end tokens for that specific project ([how to create project API tokens](../how-to/how-to-create-project-api-tokens.mdx)).
```